charts/warden/templates/configmap.yaml

```diff
@@ -1,7 +1,33 @@
+{{- $allowedRegistries := .Values.global.config.data.notary.allowedRegistries }}
+{{- if and .Values.global.config.data.notary.additionalAllowedRegistries .Values.global.config.data.notary.defaultAllowedRegistries -}}
+{{- $allowedRegistries = join ", " (concat .Values.global.config.data.notary.defaultAllowedRegistries  .Values.global.config.data.notary.additionalAllowedRegistries | uniq ) }}
```

Comment:
Is it wanted that in this line we override the original value of the `allowedRegistries` field?

Reply:
Yes. here I overwrite  the `allowedRegistries` variable in case `defaultAllowedRegistries ` values is given, This (plus `additionalAllowedRegistries`) should overrule the deprecated `allowedRegistries` value